src/common/SPELLS/MAGE.js

```diff
@@ -209,11 +221,6 @@ export default {
     name: 'Shatter',
     icon: 'spell_frost_frostshock',
   },
-  FREEZE: {
-    id: 231596,
```

Comment:
What's with this Freeze being different id / icon than the one added above?

Reply:
I believe the old ID (231596) was incorrect and the new one (33395) is the correct one. 

This is the old one (which looks like it is the passive that gives the elemental the ability to use freeze)
http://www.wowhead.com/spell=231596/freeze

and this is the new one, which looks like the actual freeze ability
http://www.wowhead.com/spell=33395/freeze

Reply:
Have you confirmed this with logs? I know right now it doesn't matter because none of our code uses the SPELLS entry, but later on when someone does use it I don't want them surprised when it doesn't work.

Reply:
I havent, but ill go digging through warcraft logs to see if i can find a log where someone didnt use lonely winter and used freeze .... that should be fun.

Reply:
If you can't find instances anywhere in logs, that's OK, just remove the FREEZE entry entirely from SPELLS. I think it's a good rule of thumb that you shouldn't add something to SPELLS until you've seen that it actually shows as you expect in logs.

Reply:
The icon of the new ID matches the icon in game, so im almost positive that that is the correct ID, but ill still remove it for now.